src/Field/Configurator/CountryConfigurator.php

```
@@ -44,6 +46,10 @@ private function getCountryName(?string $countryCode): ?string
         }
 
         try {
+            if ($alpha3) {
```

Please also add this to the fallback version (before 4.3) above.

---

It's available since Symfony 4.4/5.0, so it wouldn't make sense.

---

Ah I see.. but in that case maybe we should throw a helpful error message if `alpha3` option is set, but it's not supported?

---

I don't have any project using Symfony < 5.1 anymore so I can't be sure, but I think Symfony Form will complain about this option not being available on CountryType.

Do you think it's worth checking it & throwing a custom exception?

---

Mh.. not sure to be honest.. lets see what Javier has to say when he takes a look 🙂

---

@javiereguiluz What do you think?

---

I agree with @jmsche because all our composer.json Symfony dependencies are `^4.4 || ^5.0`, so `4.3` is not supported (at least officially) and then we can use this option because was introduced in Symfony 4.4.